lib/streamlit/elements/image.py

```diff
@@ -164,15 +168,22 @@ def _image_may_have_alpha_channel(image: PILImage) -> bool:
         return False
 
 
+def _image_is_gif(image: PILImage) -> bool:
+    if image.format == "GIF":
```

Comment:
nit: You can simplify this to:
```python
return image.format == "GIF"
```

Reply:
I reverted the change because `return image.format == "GIF"` returns type `Any`, not `bool`, and mypy complains

Reply:
> Probably works if you wrap it into a bool(image.format == "GIF")

That worked!